src/register/OptionalFields.jsx

```diff
@@ -1,110 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-
-import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
-import { getConfig } from '@edx/frontend-platform';
-import { Form, Icon } from '@edx/paragon';
-import { ExpandMore } from '@edx/paragon/icons';
-
-import { EDUCATION_LEVELS, GENDER_OPTIONS, YEAR_OF_BIRTH_OPTIONS } from './data/constants';
-import messages from './messages';
-
-const OptionalFields = (props) => {
```

Comment:
Instead of removing, can we utilize these on the progressive profiling page in some way?

Reply:
We will probably not need this code for our approach but the reason for creating this PR was to remove the fields from Registration folder. These are already duplicated in progressive profiling folder.